package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@r/api-client",
-  "version": "1.23.5",
+  "version": "1.23.6",
```

Comment:
nit: let's make this `1.24.0`, since it's a more impactful change than just a bugfix.